src/WingetCreateCore/Common/PackageParser.cs

```diff
@@ -209,7 +233,24 @@ public static bool UpdateInstallerNodes(InstallerManifest installerManifest, IEn
                 matchingExistingInstaller.Platform = newInstaller.Platform;
             }
 
-            return true;
+            return !installerMissingMatch.Any();
+        }
+
+        /// <summary>
+        /// Performs a regex match to determine the installer architecture based on the url string.
+        /// </summary>
+        /// <param name="url">Installer url string.</param>
+        /// <returns>Installer architecture enum.</returns>
+        private static InstallerArchitecture? GetArchFromUrl(string url)
```

Comment:
>GetArchFromUrl [](http://example.com/codeflow?start=46&length=14)

Use new logic from linqpad script

Reply:
Changed to newest logic from linqpad script